src/docs/user/references/Notification-service.adoc

```diff
@@ -124,15 +125,16 @@ section in the Admin guide to look at the full configuration.
 ===== Notification Method confirmation
 
 An administrator can activate confirmation request for notification methods other than portal.
-In such case, after providing the notification method parameter (Email, Third-party parameter, Telephone)
-, an input field will appear below the parameter asking for a confirmation code.
+In such case, after providing the notification method parameter, an input field will appear below the third-party parameter or telephone parameter asking for a confirmation code.
```

Comment:
In such a scenario, once you've specified the notification method parameter, an input field will become visible below the third-party parameter or telephone parameter, prompting the user to enter a confirmation code. In the case of emails, the verification code input field is always shown.